.github/workflows/update-bidi-types.yml

```diff
@@ -0,0 +1,68 @@
+# This workflow will update the WebdriverBidi types.
+
+name: Update WebdriverBidi types
+
+on:
+  schedule:
+    # Run hourly at xx:30.
+    - cron: '30 * * * *'
```

Comment:
Too spammy, types change infrequently and we have a manual trigger so I would put it at 1 time per day.

Reply:
Once per day seems reasonable, it would align with WPT. Set it to midnight, maybe.